src/main/resources/META-INF/plugin.xml

```diff
@@ -12,6 +12,12 @@
     ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>1.4.3</b></p>
+  <ul>
+    <li>This extension is now deprecated.</li>
```

Comment:
We do not need this line as we are mentioning at the top this plugin is deprecated.